app/components/assets/asset-qr.tsx

```diff
@@ -0,0 +1,173 @@
+          downloadLink.click();
+
+          // Clean up the object URL after the download
+          URL.revokeObjectURL(downloadLink.href);
+        })
+        // eslint-disable-next-line no-console
+        .catch(console.error);
+    }
+  }
+
+  const printQr = useReactToPrint({
+    content: () => captureDivRef.current,
+  });
+  return (
+    <div className="">
+      <div className="mb-4 flex w-auto justify-center rounded-xl border border-solid p-6">
+        <QrLabel ref={captureDivRef} data={qrObj} title={asset.title} />
+      </div>
+      <ul className="description-list">
+        {/* <li className="mb-4 flex justify-between text-gray-600">
+            <label
+              htmlFor="size"
+              className="key max-w-[120px] break-words font-medium"
+            >
+              Size
+            </label>
+            <span className="value max-w-[190px] break-words font-semibold">
+              <Form method="get" ref={formRef}>
+                <select
+                  name="size"
+                  value={data.qr.size}
+                  onChange={handleSizeChange}
+                  className=" border-none py-0 pr-6"
+                  style={{ backgroundPosition: "right center" }}
+                >
+                  {Object.keys(data.sizes).map((size) => (
+                    <option key={size} value={size}>
+                      {size}
+                    </option>
+                  ))}
+                </select>
+              </Form>
+            </span>
+          </li> */}
+        {/* <li className="mb-4 flex justify-between text-gray-600">
+            <span className="key max-w-[120px] break-words font-medium">
+              File
+            </span>
+            <span className="value max-w-[190px] break-words font-semibold">
+              PNG
+            </span>
+          </li> */}
+      </ul>
+      {/* using this button to convert html to png and download image using the a tag below */}
+      <div className="flex items-center gap-3">
+        <Button
+          icon="download"
+          onClick={downloadQr}
+          download={`${slugify(asset.title)}-${qrObj.qr.size}-shelf-qr-code-${
+            qrObj.qr.id
+          }.png`}
+          ref={downloadQrBtnRef}
+          variant="secondary"
+          className="w-full"
+        >
+          Download
+        </Button>
+        <Button
+          icon="print"
+          variant="secondary"
+          className="w-full"
+          onClick={printQr}
+        >
+          Print
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+interface QrLabelProps {
+  data: {
+    qr: {
+      id: string;
+      size: SizeKeys;
+      src: string;
+    };
+  };
+  title: string;
+}
+
+const QrLabel = React.forwardRef<HTMLDivElement, QrLabelProps>((props, ref) => {
+  const { data, title } = props;
+  return (
+    <div
+      className="flex h-auto w-4/5 flex-col justify-center gap-3 rounded-md border-[5px] border-[#E3E4E8] bg-white px-8 py-[17px]"
```

Comment:
The `w-4/5` as far as I remember, was specifically done because the QR panel had a fixed width so it was calculated based on that. Please make the column of the qr have the same width as the QR sidebar had before.